app/components/index-info.js

```diff
@@ -34,7 +34,12 @@ export default Component.extend({
 
     let self = this;
     promise.then(function(value) {
+      // Why do we need to do this?  Why is self 'destroyed'.
```

Comment:
is this a new thing?

Reply:
Just avoiding a problem trying to do an operation on an object that has been destroyed.  Leaving the check.